packages/react-components/package.json

```diff
@@ -51,6 +51,7 @@
     "@fluentui/react-theme": "^9.0.0-alpha.22",
     "@fluentui/react-tooltip": "^9.0.0-alpha.75",
     "@fluentui/react-utilities": "^9.0.0-alpha.43",
+    "storybook-addon-export-to-codesandbox": "^0.1.8",
```

Comment:
I guess to that this go a root `package.json` and be without a caret:

```
    "storybook-addon-export-to-codesandbox": "0.1.8",
```

(devDependencies are installed into root `package.json`)